recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
Ok, so am not too worried about these ATM. Though I think these are all things that could be raised at the feedstock. Here were a few [thoughts]( https://github.com/conda-forge/staged-recipes/pull/1678#issuecomment-250269657 ) along these lines.